homeWork2/getArrayPokemons.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+async function getArrayPokemons() {
+  const response = await fetch(`https://pokeapi.co/api/v2/pokemon`);
+  const objectResults = await response.json();
```

Comment:
`const { results } = await response.json()` . This will allow you to remove line 6